packages/agoric-cli/src/lib/chain.js

```diff
@@ -43,6 +43,7 @@ harden(normalizeAddressWithOptions);
  * @param {MinimalNetworkConfig & {
  *   from: string,
  *   fees?: string,
+ *   gas?: number | 'auto' | ['auto', adjustment?: number | undefined],
```

Comment:
this is quite complicated. please factor into an arg parsing helper

Reply:
Done.